reconcile/gql_definitions/common/saas_files.gql

```diff
@@ -0,0 +1,177 @@
+# qenerate: plugin=pydantic_v1
+
+query SaasFiles {
+  saas_files: saas_files_v2 {
+    path
+    name
+    app {
+      name
+    }
+    pipelinesProvider {
+      name
+      provider
+      ... on PipelinesProviderTekton_v1 {
+        namespace {
+          name
+          cluster {
+            ...OcConnectionCluster
+            consoleUrl
+          }
+        }
+        defaults {
+          pipelineTemplates {
+            openshiftSaasDeploy {
+              name
+            }
+          }
+        }
+        pipelineTemplates {
+          openshiftSaasDeploy {
+            name
+          }
+        }
+      }
+    }
+    deployResources {
+      requests {
+        cpu
+        memory
+      }
+      limits {
+        cpu
+        memory
```

Comment:
nit: I think we have a fragment for these https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/gql_definitions/fragments/resource_values.gql

Reply:
Unfortunately, the fragment is of type `ResourceValues_v1`, and this type is `ResourceRequirements_v1`. Both are very similar, but the attributes are required in `ResourceRequirements_v1`.